coreos-ostree-importer/coreos_ostree_importer.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/python3
+import fedora_messaging
+import fedora_messaging.api
+
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import traceback
+import tempfile
+import os
+import tarfile
+import hashlib
+
+
+import botocore
+import boto3
+
+# from fedora_messaging import message
+# from fedora_messaging.api import publish
+# from fedora_messaging.config import conf
+
+# Set local logging
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+FEDORA_MESSAGING_TOPIC_LISTEN = (
+    "org.fedoraproject.prod.coreos.build.request.ostree-import"
+)
+FEDORA_MESSAGING_TOPIC_RESPOND = FEDORA_MESSAGING_TOPIC_LISTEN + ".finished"
+
+
+# We are processing the org.fedoraproject.prod.coreos.build.request.ostree-import topic
+# https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.coreos.build.request.ostree-import&delta=100000
+EXAMPLE_MESSAGE_BODY = json.loads(
```

Comment:
Probably could throw a link to https://github.com/coreos/fedora-coreos-tracker/issues/198#issuecomment-513944390 here, though will try to throw that up as a PR.

Reply:
You mean to link to that comment rather than having the `EXAMPLE_MESSAGE_BODY` variable? I `EXAMPLE_MESSAGE_BODY` when testing in the `main()`.

Reply:
> You mean to link to that comment rather than having the EXAMPLE_MESSAGE_BODY variable?

No, just adding it as a comment to give a linkback with more context on where the schema comes from.

Reply:
added a comment with the link!